### PR TITLE
Drop symfony 4.3 support

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes -
| Related issues/PRs | part of #4798 
| License | MIT
| Documentation PR | sulu/sulu-docs#prnum

#### What's in this PR?

Drop symfony 4.3 support.

#### Why?

To make later the update from 4.4 to 5.0 seamless.

#### To Do

- [ ] ~~Create a documentation PR~~
- [ ] ~~Add breaking changes to UPGRADE.md~~

### CODE REVIEW

**Comment by danrot** [comment]
@alexander-schranz Looks like it is your task to fix phpstan :see_no_evil:

**Comment by danrot** [comment]
@alexander-schranz There are still a few errors when being run with `--prefer-lowest`...

**Comment by alexander-schranz** [comment]
@danrot was an error in prophecy with nullable types. Did set the min prophecy version from 1.7 to 1.8.

File: composer.json [hunk 1]
```diff
@@ -61,45 +61,46 @@
         "pagerfanta/pagerfanta": "^1.0.4 || ^2.0",
         "php-http/client-common": "^1.1.0 || ^2.0",
         "php-http/guzzle6-adapter": "^1.1 || ^2.0",
+        "phpspec/prophecy": "^1.8",
```
Comment (danrot) [inline composer.json:64]: Should still be a dev-dependency.

Comment (alexander-schranz) [reply composer.json]: Sure 🙈

**Pull request merged**